Cargo.toml

```diff
@@ -16,6 +16,8 @@ x86 = "0.41"
 log = "0.4"
 lazy_static = { version = "1.4", features = ["spin_no_std"] }
 cstr_core = { version = "0.2.3", default-features = false }
+bitflags = "1.2.1"
+uefi = "0.11.0"
```

Comment:
Can we get rid of this dependency by embedding the 3x  types {MemoryAttribute, MemoryDescriptor, MemoryType} in the topology crate? It seems overkill and will simplify our future lifes since we have to update this crate whenever we update the uefi version in the kernel if we too if we include the dependency here.

Reply:
Done